Building DAG of jobs...
Using shell: /bin/bash
Provided cores: 36
Rules claiming more threads will be scaled down.
Job counts:
	count	jobs
	2	STAR_align
	1	all
	2	count_reads
	2	tx_sort
	7

[Fri Nov  6 13:46:41 2020]
rule STAR_align:
    input: results/Metadata/SRR12845350.json, results/Rawdata/Fastq/SRR12845350_1.fastq, results/Rawdata/Fastq/SRR12845350_2.fastq
    output: results/Rawdata/Aligned/SRR12845350_Aligned.sortedByCoord.out.bam, results/Rawdata/Aligned/SRR12845350_Aligned.toTranscriptome.out.bam, results/Rawdata/Aligned/SRR12845350_Chimeric.out.junction, results/log/SRR12845350_Log.final.out, results/log/SRR12845350_Log.out, results/log/SRR12845350_Log.progress.out, results/Rawdata/Aligned/SRR12845350_SJ.out.tab
    log: info/logs/SRR12845350.star_align.log
    jobid: 4
    benchmark: info/benchmark/SRR12845350.star_align.bench
    wildcards: sample=SRR12845350
    threads: 4

[Fri Nov  6 13:46:41 2020]
rule STAR_align:
    input: results/Metadata/SRR8094770.json, results/Rawdata/Fastq/SRR8094770_1.fastq, results/Rawdata/Fastq/SRR8094770_2.fastq
    output: results/Rawdata/Aligned/SRR8094770_Aligned.sortedByCoord.out.bam, results/Rawdata/Aligned/SRR8094770_Aligned.toTranscriptome.out.bam, results/Rawdata/Aligned/SRR8094770_Chimeric.out.junction, results/log/SRR8094770_Log.final.out, results/log/SRR8094770_Log.out, results/log/SRR8094770_Log.progress.out, results/Rawdata/Aligned/SRR8094770_SJ.out.tab
    log: info/logs/SRR8094770.star_align.log
    jobid: 7
    benchmark: info/benchmark/SRR8094770.star_align.bench
    wildcards: sample=SRR8094770
    threads: 4

Activating conda environment: /home/kasper/Git_repositories/LiverDB/.snakemake/conda/fd596e9f
Activating conda environment: /home/kasper/Git_repositories/LiverDB/.snakemake/conda/fd596e9f
Terminating processes on user request, this might take some time.
[Fri Nov  6 13:46:56 2020]
Error in rule STAR_align:
    jobid: 4
    output: results/Rawdata/Aligned/SRR12845350_Aligned.sortedByCoord.out.bam, results/Rawdata/Aligned/SRR12845350_Aligned.toTranscriptome.out.bam, results/Rawdata/Aligned/SRR12845350_Chimeric.out.junction, results/log/SRR12845350_Log.final.out, results/log/SRR12845350_Log.out, results/log/SRR12845350_Log.progress.out, results/Rawdata/Aligned/SRR12845350_SJ.out.tab
    log: info/logs/SRR12845350.star_align.log (check log file(s) for error message)
    conda-env: /home/kasper/Git_repositories/LiverDB/.snakemake/conda/fd596e9f

RuleException:
CalledProcessError in line 172 of /home/kasper/Git_repositories/LiverDB/workflow/Snakefile.py:
Command 'source /home/kasper/miniconda3/bin/activate '/home/kasper/Git_repositories/LiverDB/.snakemake/conda/fd596e9f'; set -euo pipefail;  /home/kasper/miniconda3/bin/python3.7 /home/kasper/Git_repositories/LiverDB/.snakemake/scripts/tmpq8sai0fu.star_align.py' returned non-zero exit status 1.
  File "/home/kasper/Git_repositories/LiverDB/workflow/Snakefile.py", line 172, in __rule_STAR_align
  File "/home/kasper/miniconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job STAR_align since they might be corrupted:
results/Rawdata/Aligned/SRR12845350_Aligned.sortedByCoord.out.bam, results/Rawdata/Aligned/SRR12845350_Aligned.toTranscriptome.out.bam
[Fri Nov  6 13:46:56 2020]
Error in rule STAR_align:
    jobid: 7
    output: results/Rawdata/Aligned/SRR8094770_Aligned.sortedByCoord.out.bam, results/Rawdata/Aligned/SRR8094770_Aligned.toTranscriptome.out.bam, results/Rawdata/Aligned/SRR8094770_Chimeric.out.junction, results/log/SRR8094770_Log.final.out, results/log/SRR8094770_Log.out, results/log/SRR8094770_Log.progress.out, results/Rawdata/Aligned/SRR8094770_SJ.out.tab
    log: info/logs/SRR8094770.star_align.log (check log file(s) for error message)
    conda-env: /home/kasper/Git_repositories/LiverDB/.snakemake/conda/fd596e9f

RuleException:
CalledProcessError in line 172 of /home/kasper/Git_repositories/LiverDB/workflow/Snakefile.py:
Command 'source /home/kasper/miniconda3/bin/activate '/home/kasper/Git_repositories/LiverDB/.snakemake/conda/fd596e9f'; set -euo pipefail;  /home/kasper/miniconda3/bin/python3.7 /home/kasper/Git_repositories/LiverDB/.snakemake/scripts/tmpv69_plhx.star_align.py' returned non-zero exit status 1.
  File "/home/kasper/Git_repositories/LiverDB/workflow/Snakefile.py", line 172, in __rule_STAR_align
  File "/home/kasper/miniconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job STAR_align since they might be corrupted:
results/Rawdata/Aligned/SRR8094770_Aligned.sortedByCoord.out.bam, results/Rawdata/Aligned/SRR8094770_Aligned.toTranscriptome.out.bam
Complete log: /home/kasper/Git_repositories/LiverDB/workflow/.snakemake/log/2020-11-06T134640.976898.snakemake.log
